worksheet_B/readme.MD

```diff
@@ -0,0 +1,38 @@
+code:
+```
+
+START
+
+Attempts=0
+i=0
+Likeness=0
```

Comment:
This needs to be done inside the while loop, otherwise the likeness score will accumulate between guesses.